packages/cardhost/app/lib/card-model-for-browser.ts

```diff
@@ -90,22 +92,27 @@ export default class CardModelForBrowser implements CardModel {
     }
   }
 
-  // TODO: add failing test for `_id` being unimplemented here and then fix
-  async adoptIntoRealm(realm: string, _id?: string): Promise<CardModel> {
+  async adoptIntoRealm(realm: string, id?: string): Promise<CardModel> {
     if (this.state.type !== 'loaded') {
       throw new Error(`tried to adopt from an unsaved card`);
     }
+    let builder = await this.cards.builder();
```

Comment:
This does a lot of work that I don't think we need to do necessarily. The realmURL is a constant exported by the builder that is what your checking when you do builder.realmURL.